zookeeper.go

```diff
@@ -0,0 +1,105 @@
+package rollout
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/samuel/go-zookeeper/zk"
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.New()
```

Comment:
If PT has some existing code conventions around using `log` as a package variable for a logrus instance, that's fine. But otherwise I think this might be a confusing name, as I presupposed that uses of `log.Something(...)` were invocations of the standard library's `log` package.